renovate.json

```diff
@@ -30,11 +30,14 @@
         "org.robolectric",
         "com.google.truth",
         "com.google.testparameterinjector",
-        "androidx.test"
+        "androidx.test",
+        "kotlin",
+        "ksp"
       ],
       "matchUpdateTypes": ["major"],
       "commitMessagePrefix": "test(deps):",
-      "commitMessageAction": "update"
+      "commitMessageAction": "update",
+      "_comment": "Need wait for this issue to be fixed https://github.com/jeremymailen/kotlinter-gradle/issues/414. After that we could remove 'kotlin', 'ksp' from this list."
```

Comment:
Related to [jeremymailen/kotlinter-gradle#414](https://github.com/jeremymailen/kotlinter-gradle/issues/414).  

I’ve disabled the automatic version increment for `kotlin` and `ksp` until the issue is resolved.

Reply:
@cre8ivejp please help me to take a look

Reply:
hi @cre8ivejp , my changes are no longer needed. 
Please merge this [PR](https://github.com/bucketeer-io/android-client-sdk/pull/209) first.
after that request renovate to rebase this PR and all will be good.

Reply:
@duyhungtnn, I just merged.